source/codegen/templates/service_helpers.mako

```diff
@@ -217,8 +217,6 @@ ${initialize_standard_input_param(function_name, parameter)}\
 %>\
 % if c_type in ['ViConstString', 'const char[]']:
       auto ${parameter_name} = ${request_snippet}.c_str();\
-% elif c_type == 'ViString' or c_type == 'ViRsrc':
```

Comment:
Note: this case is handled by the `common_helpers.is_string_arg` case below. That case is more general because it can deal with the case where `c_type` is an array (in which case we need to cast to pointer type)